### PR TITLE
Rake tasks config and beeline

### PR DESCRIPTION
**Pull request by nowNick**
Added and fixed rake tasks:

- one config file in config directory
- beeline rake task
- load hive_udfs task

### CODE REVIEW

File: lib/rspec/hive/rake_tasks/docker.rake [hunk 1]
```diff
@@ -63,14 +61,60 @@ namespace :spec do
...
+        interpolated = ERB.new(File.read(config_filepath)).result
+        config = YAML.load(interpolated)['hive']
+
+        fail 'Please provide UDFS_PATH'.red unless udfs_path
+        if udfs_path.start_with?('s3://')
+          puts 'Downloading from s3...'.yellow
+          cmd = "aws s3 ls #{udfs_path}"
+
+          fail 'awscli is not configured.'.red unless system(cmd)
+          cmd = "aws s3 cp #{udfs_path} #{config['host_shared_directory_path']}/hive-udfs.jar"
```
Comment (mknapik) [inline lib/rspec/hive/rake_tasks/docker.rake]: `File.join`

**Comment by mknapik** [comment]
Good job!

File: lib/rspec/hive/rake_tasks/docker.rake [hunk 2]
```diff
@@ -63,14 +61,60 @@ namespace :spec do
...
+          system(cmd)
+        else
+          puts 'Copying from local directory...'.yellow
+          cmd = "cp #{udfs_path} #{config['host_shared_directory_path']}/hive-udfs.jar"
+        end
+        puts 'Done'.green
+
+
```
Comment (jekuta) [inline lib/rspec/hive/rake_tasks/docker.rake]: empty line:)

File: lib/rspec/hive/rake_tasks/docker.rake [hunk 3]
```diff
@@ -63,14 +61,60 @@ namespace :spec do
...
+        docker_conatiners = `docker ps`.lines
+        if docker_conatiners.size != 2
+          raise 'There is more than 1 instance of docker container running (or no running docker containers). '\
+                'Check `docker ps` and stop containers that are not in use right now or specify CONTAINER_ID and run this command again.'.red
+        else
+          docker_conatiners[1].split[0]
+        end
+      end
+
+      desc 'Load Hive UDFS (user defined functions) onto docker'
```
Comment (jekuta) [inline lib/rspec/hive/rake_tasks/docker.rake]: with `.` at the end of the line

**Comment by jekuta** [comment]
LGMT!

**Comment by nowNick** [comment]
I have applied comments. Could you please check again?

**Comment by jekuta** [comment]
LGBT!

**Pull request merged**